boris-http/bin/dev-ghci

```diff
@@ -25,7 +25,7 @@ export BORIS_SINGLE_PROJECT_NAME=demo
 export BORIS_SINGLE_PROJECT_REPOSITORY="$LOCAL_GIT"
 
 export BORIS_STORE=postgres
-export BORIS_POSTGRES="dbname=boris_test host=docker.for.mac.localhost user=boris_test password=boris_test port=5432"
+export BORIS_POSTGRES="dbname=boris_test host=localhost user=boris_test password=boris_test port=5432"
```

Comment:
Guess we should be passing in this `host` value in the environment

Reply:
Maybe just while DB string.